packages/workbox-core/models/messages/messages.mjs

```diff
@@ -93,4 +104,24 @@ export default {
       `setCacheDeatils({${cacheNameId}: '...'}). Received a value of ` +
       `'${JSON.stringify(value)}'`;
   },
+
+  'unregister-route-but-not-found-with-method': ({route, method}) => {
+    if (!method || !route) {
+      throw new Error(`Unexpected input to ` +
+        `'unregister-route-but-not-found-with-method' error.`);
+    }
+
+    return `The route you're trying to unregister, ${route}, was not ` +
```

Comment:
Not blocking, but it's unclear with `route` is so I'm unsure what this would print. Does the string value of `route` always produce something meaningful?

Reply:
Good point.